version.js

```diff
@@ -190,14 +191,28 @@ if (program.debug) {
 	console.log(`will update ${files}`);
 }
 
-function check(source, dep, version) {
+function check(source, dep, version, category = 'dep') {
```

Comment:
this does not only check, it also mutate the source parameter, can you please split check and transform in two different function ? can you please add a bit of code doc

Reply:
I think Rename it could be nice in fact ! true